testsuite/features/build_validation/add_custom_repositories/add_rocky8_repositories.feature

```diff
@@ -1,65 +1,65 @@
 # Copyright (c) 2021-2022 SUSE LLC
 # Licensed under the terms of the MIT license.
 
-@centos8_minion
-Feature: Add the CentOS 8 distribution custom repositories
-  In order to use CentOS 8 channels with Red Hat "modules"
+@rocky8_minion
+Feature: Add the Rocky 8 distribution custom repositories
+  In order to use Rocky 8 channels with Red Hat "modules"
   As a SUSE Manager administrator
   I want to filter them out to remove the modules information
 
-  Scenario: Download the iso of CentOS 8 DVD and mount it on the server
-    When I mount as "centos-8-iso" the ISO from "http://minima-mirror-bv.mgr.prv.suse.net/pub/centos/8/isos/x86_64/CentOS-8.2.2004-x86_64-dvd1.iso" in the server
+  Scenario: Download the iso of Rocky 8 DVD and mount it on the server
+    When I mount as "rocky-8-iso" the ISO from "http://minima-mirror-bv.mgr.prv.suse.net/pub/rocky/8/isos/x86_64/Rocky-x86_64-dvd1.iso" in the server
 
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
-  Scenario: Add a child channel for CentOS 8 DVD repositories
+  Scenario: Add a child channel for Rocky 8 DVD repositories
     When I follow the left menu "Software > Manage > Channels"
     And I follow "Create Channel"
-    And I enter "Custom Channel for CentOS 8 DVD" as "Channel Name"
-    And I enter "centos-8-iso" as "Channel Label"
+    And I enter "Custom Channel for Rocky 8 DVD" as "Channel Name"
+    And I enter "rocky-8-iso" as "Channel Label"
     And I select "RHEL8-Pool for x86_64" from "Parent Channel"
     And I enter "Custom channel" as "Channel Summary"
     And I click on "Create Channel"
-    Then I should see a "Channel Custom Channel for CentOS 8 DVD created" text
+    Then I should see a "Channel Custom Channel for Rocky 8 DVD created" text
 
-  Scenario: Add the CentOS 8 Appstream DVD repository
+  Scenario: Add the Rocky 8 Appstream DVD repository
     When I follow the left menu "Software > Manage > Repositories"
     And I follow "Create Repository"
-    And I enter "centos-8-iso-appstream" as "label"
-    And I enter "http://127.0.0.1/centos-8-iso/AppStream" as "url"
+    And I enter "rocky-8-iso-appstream" as "label"
+    And I enter "http://127.0.0.1/rocky-8-iso/AppStream" as "url"
```

Comment:
Did you confirmed that it follows the same paths "AppStream" and "BaseOS" ?

Reply:
Yes.
```bash
❯ sudo mount Rocky-8-x86_64-dvd1.iso /mnt
[sudo] password for root: 
mount: /mnt: WARNING: source write-protected, mounted read-only.

❯  ll /mnt/
total 14
dr-xr-xr-x 4 root root 2048 2022-05-15 23:06 AppStream
dr-xr-xr-x 4 root root 2048 2022-05-15 23:06 BaseOS
dr-xr-xr-x 3 root root 2048 2022-05-15 23:06 EFI
dr-xr-xr-x 3 root root 2048 2022-05-15 23:06 images
dr-xr-xr-x 2 root root 2048 2022-05-15 23:06 isolinux
-r--r--r-- 1 root root 2204 2022-03-30 02:02 LICENSE
-r--r--r-- 1 root root   86 2022-05-15 23:05 media.repo
-r--r--r-- 1 root root  883 2022-05-15 23:06 TRANS.TBL

❯  ll /mnt/AppStream/
total 8
dr-xr-xr-x 29 root root 4096 2022-05-15 23:06 Packages
dr-xr-xr-x  2 root root 4096 2022-05-15 23:06 repodata
dgedon@b660:~/isos> ll /mnt/BaseOS/
total 8
dr-xr-xr-x 28 root root 4096 2022-05-15 23:06 Packages
dr-xr-xr-x  2 root root 4096 2022-05-15 23:06 repodata
```